### PR TITLE
Modify some code in eventbus module.

### PR DESCRIPTION
**Pull request by gy95**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:NONE

```release-note

```

### CODE REVIEW

File: edge/pkg/eventbus/common/util/common.go [hunk 1]
```diff
@@ -66,7 +66,7 @@ func HubClientInit(server, clientID, username, password string) *MQTT.ClientOpti
 
 		caCert, err := ioutil.ReadFile(eventconfig.Config.TLS.TLSMqttCAFile)
 		if err != nil {
-			klog.Errorf("Failed to read TLSMqttCAFile")
+			klog.Errorf("Failed to read TLSMqttCAFile: %v", err)
```
Comment (fisherxu) [inline edge/pkg/eventbus/common/util/common.go:69]: Let's use lowercase for all the log, ref: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

File: edge/pkg/eventbus/mqtt/server.go [hunk 2]
```diff
@@ -107,10 +108,13 @@ func (m *Server) onSubscribe(msg *packet.Message) {
 	// for "SYS/dis/upload_records", no need to base64 topic
 	var target string
 	resource := base64.URLEncoding.EncodeToString([]byte(msg.Topic))
-	if strings.HasPrefix(msg.Topic, "$hw/events/device") || strings.HasPrefix(msg.Topic, "$hw/events/node") {
+	if strings.HasPrefix(msg.Topic, dtcommon.DeviceETPrefix) || strings.HasPrefix(msg.Topic, dtcommon.MemETPrefix) {
```
Comment (fisherxu) [inline edge/pkg/eventbus/mqtt/server.go:111]: One mudule shouldn't depend on another, we need to move these const to a common place.

File: edge/pkg/eventbus/common/util/common.go (hunk 1 above)
Comment (gy95) [reply edge/pkg/eventbus/common/util/common.go:69]: ok，modificaion is complete

File: edge/pkg/eventbus/eventbus.go [hunk 3]
```diff
@@ -117,30 +117,36 @@ func (eb *eventbus) pubCloudMsgToEdge() {
...
+				continue
 			}
 			message := body["message"].(map[string]interface{})
 			topic := message["topic"].(string)
-			payload, _ := json.Marshal(&message)
-			eb.publish(topic, payload)
+			payload, err := json.Marshal(&message)
+			if err != nil {
+				eb.publish(topic, payload)
+			}
```
Comment (fisherxu) [inline edge/pkg/eventbus/eventbus.go:127]:
```suggestion
			if err != nil {
				klog.Errorf("...")
				continue
			}
			eb.publish(topic, payload)
```

File: edge/pkg/eventbus/eventbus.go [hunk 4]
```diff
@@ -117,30 +117,36 @@ func (eb *eventbus) pubCloudMsgToEdge() {
...
-				return
+				continue
 			}
 			topic := fmt.Sprintf("$hw/events/node/%s/authInfo/get/result", eventconfig.Config.NodeName)
-			payload, _ := json.Marshal(accessInfo.GetContent())
-			eb.publish(topic, payload)
+			payload, err := json.Marshal(accessInfo.GetContent())
+			if err != nil {
+				eb.publish(topic, payload)
+			}
```
Comment (fisherxu) [inline edge/pkg/eventbus/eventbus.go:149]: ditto

File: edge/pkg/eventbus/mqtt/server.go [hunk 5]
```diff
@@ -111,6 +111,9 @@ func (m *Server) onSubscribe(msg *packet.Message) {
 		target = modules.TwinGroup
 	} else {
 		target = modules.HubGroup
+		if msg.Topic == UploadTopic {
+			resource = UploadTopic
+		}
```
Comment (fisherxu) [inline edge/pkg/eventbus/mqtt/server.go:116]: Why do we need it?

**Pull request closed without merging**